panel/src/components/View/Buttons/ThemeButton.vue

```diff
@@ -1,14 +1,20 @@
 <template>
 	<k-button
+		v-if="$panel.view.id === 'account'"
```

Comment:
Could there be any other view where the users might want to include the theme button?

Reply:
In theory this button could live on any view as it does not depend on any props. But since our user and account view are kind of the same, then it would by default show up on any user view but always only relate to the current user, which could be quite confusing.